server/routes/api/index.js

```diff
@@ -0,0 +1,38 @@
+const apiRouter = require('express').Router();
+const { Octokit } = require('@octokit/rest');
+const debug = require('../../debug');
+
+const octokit = Octokit({
```

Comment:
Well, the `Router` example from `express` docs does all the job at the module scope, but it's a bad pattern and should be used only for the sake of simplicity in examples. We can inherit the `Router` class and create proper methods for route handlers.